torch/_inductor/codegen/simd.py

```diff
@@ -115,6 +231,12 @@ def __init__(
         self.kernel = kernel
         self.root = root
 
+    @property
+    @cache_on_self
+    @no_type_check
```

Comment:
I thought mypy would break here, but it seems like it's now working. Maybe this changed in the most recent version. Removed this in https://github.com/pytorch/pytorch/pull/141738.